### PR TITLE
Temporarily fix the end-to-end training test

### PR DESCRIPTION
**Pull request by Leon-Leyang**
## Issue ticket number and link

--

## What are the changes

Modified the code to upload a model during the init of `RModelWrapper` to pass the end-to-end training test temporarily.

## Comments

- Should remove the logic of initializing the model and uploading it during the init of `RModelWrapper` in the future. Uploading a model and then training it should be an elegant way for the e2e test.

## Checklist before requesting a review

- [x] Double check the diff between my branch and the target branch. Make sure there's nothing unexpected.
- [x] Tag relevant issues and PRs in the first section of this form.
- [x] All CI tests are passed.
- [x] No `package-lock.json` or `package.json` changes are pushed. If you do, add a new section `Dependency Changes` to the form stating what library changes are introduced and why.

### CODE REVIEW

File: back-end/objects/RModelWrapper.py [hunk 1]
```diff
@@ -52,6 +52,41 @@ def __init__(
         else:
             print("Checkpoint file not found: {}".format(net_path))
 
+        # TODO: Should remove this in the future. Currently added for passing the e2e test.
+        # Also stop passing app to this class.
+        self.upload_model_4_e2e_test(app)
+
+    def upload_model_4_e2e_test(self, app):
```
Comment (PaulCCCCCCH) [inline back-end/objects/RModelWrapper.py:60]: This is a hacky way to make things work. Let's add a todo here when model upload works end-to-end.

```suggestion
    # TODO(Chonghan): upload a model at the start of e2e tests, and remove this afterwards
    def upload_model_4_e2e_test(self, app):
```

File: back-end/objects/RModelWrapper.py [hunk 2]
```diff
@@ -52,6 +52,41 @@ def __init__(
...
+
+        metadata_4_save = {
+            "class_name": self.modelwork_type,
+            "nickname": "simple-classifier",
+            "predefined": True,
+            "pretrained": False,
+            "description": "This is the model for testing.",
+            "tags": ["test"],
+            "create_time": datetime.now(),
+            "code_path": None,
```
Comment (PaulCCCCCCH) [inline back-end/objects/RModelWrapper.py]: Just fyi, I think you can simply ignore `None` fields (and maybe `0` fields as well). They will default to null in database.

**Review by PaulCCCCCCH — APPROVED** [review]
Thanks for the fix to unblock a lot of things! 
Approved with a few comments.

File: back-end/objects/RModelWrapper.py (hunk 2 above)
Comment (Leon-Leyang) [reply back-end/objects/RModelWrapper.py]: Got it

**Pull request merged**